### ISSUE TITLE
Is there a plan to support PDF directly ?

### ISSUE THREAD

**Issue by Nokia808** [Flameeyes/unpaper#59]
I would like to ask if there a future plan to give direct support for PDF without intermidiate transformation into images during process ?

Currently, no such support available & this result in huge sized output PDF result ...

Best.

**Comment by Flameeyes** [comment]
Not really. Part of the reason is that if the PDF is generated (as in, it is not just embedded JPEG images), there should be no reason to go through unpaper.

Also, parsing PDFs is extremely complicated and I would rather not try that, or deal with poppler to do that.
